src/Libraries/Microsoft.Extensions.Http.Telemetry/Logging/HttpClientLoggingExtensions.cs

```diff
@@ -236,6 +240,19 @@ public static IServiceCollection AddHttpClientLogEnricher<T>(this IServiceCollec
         return services;
     }
 
+    private static void RemoveDefaultLogging(HttpMessageHandlerBuilder b)
+    {
+        // Remove the logger handlers added by the filter. Fortunately, they're both public, so it is a simple test on the type.
+        for (var i = b.AdditionalHandlers.Count - 1; i >= 0; i--)
+        {
+            var handlerType = b.AdditionalHandlers[i].GetType();
+            if (handlerType == typeof(LoggingScopeHttpMessageHandler) || handlerType == typeof(LoggingHttpMessageHandler))
```

Comment:
Can we cache both `typeof`s and store their values in variables?

Reply:
Why? typeof costs nothing, it compiles down to a constant value load.